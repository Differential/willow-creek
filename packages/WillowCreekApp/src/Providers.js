import { nest } from 'recompose';
import { Providers } from '@apollosproject/ui-kit';
import ClientProvider from './client';

export default nest(ClientProvider, Providers);
