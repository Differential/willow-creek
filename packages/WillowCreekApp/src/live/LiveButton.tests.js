import React from 'react';
import renderer from 'react-test-renderer';

import Providers from 'WillowCreekApp/src/Providers';

import LiveNowButton from './LiveButton';
import getLiveStream from './getLiveStream';

const mocks = [
  {
    request: {
      query: getLiveStream,
      variables: {
        isLive: true,
      },
    },
    result: {
      data: {
        liveStream: { isLive: true },
      },
    },
  },
];

it('renders without error', () => {
  const tree = renderer.create(
    <Providers mocks={mocks} addTypename={false}>
      <LiveNowButton isLive />
    </Providers>
  );
  expect(tree).toMatchSnapshot();
});
