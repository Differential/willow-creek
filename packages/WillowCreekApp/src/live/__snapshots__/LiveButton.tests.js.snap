// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without error 1`] = `
<RCTSafeAreaView
  onLayout={[Function]}
  style={
    Object {
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
  />
  <View
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</RCTSafeAreaView>
`;
